docs/customization/model.rst

```diff
@@ -0,0 +1,109 @@
+Customizing Models
+==================
+
+All models in Sylius are placed in the ``Sylius\Component\*ComponentName*\Model`` namespaces alongside with their interfaces.
+
+.. warning::
+    Many models in Sylius are **extended in the Core component**.
+    If the model you are willing to override exists in the ``Core`` your should be extending the ``Core`` one, not the base model from the component.
+
+Why would you customize a Model?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To give you an idea of some purposes of models customizing have a look at a few examples:
+
+* Add ``flag`` field to the ``Country``
+* Add ``secondNumber`` to the ``Customer``
+* Change the ``reviewSubject`` of a ``Review`` (in Sylius we have ``ProductReviews`` but you can imagine for instance a ``CustomerReview``)
```

Comment:
I don't know if it's the best example - ``reviewSubject`` of a ``Review`` needs to be ``ReviewableInterface``which is mapped dynamically. Changes of subject should be done via configuration in ``yaml`` files.